test/unit/app/models/speech_test.rb

```diff
@@ -145,8 +145,9 @@ class SpeechTest < ActiveSupport::TestCase
   end
 
   test "search_index includes default image_url if it has one" do
-    image = create(:image)
-    speech = create(:published_speech, images: [image])
+    image = build(:image)
+    speech = build(:published_speech, document: build(:document))
```

Comment:
Does the below work? Again trying to avoid magic stubbing
```suggestion
    speech = build(:published_speech, document: build(:document), lead_image: image)
```

Reply:
thanks have sorted